src/redux/store.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
+import missionReducer from './mission/missionSlice';
+import rocketReducer from './rocket/rocketSlice';
+
+const store = configureStore({
+  reducer: combineReducers({
+    missions: missionReducer,
+    rockets: rocketReducer,
+  }),
+});
+
+export default store;
```

Comment:
- [ ] **[OPTIONAL]** Based on the requirements you should have the `redux-logger` package installed in this application. Configuring redux-logger helps you monitor the store and the dispatch actions in the browser. You have done well adding the package but you failed to configure it. Here is a suggestion:

  ```
  import { combineReducers, configureStore } from '@reduxjs/toolkit';
  import logger from 'redux-logger';
  import missionReducer from './mission/missionSlice';
  import rocketReducer from './rocket/rocketSlice';
  
  const store = configureStore({
    reducer: combineReducers({
      missions: missionReducer,
      rockets: rocketReducer,
    }),
    middleware: (getDefaultMiddleware) => getDefaultMiddleware().concat(logger),
  });
  
  export default store;
  ```